nilearn/__init__.py

```diff
@@ -9,66 +9,21 @@
 See http://nilearn.github.com for complete documentation.
 """
 
-from version import __version__
+from .version import _check_module_dependencies, __version__
 
 
-def _import_module_with_version_check(module_name, minimum_version):
-    """Check that module is installed with a recent enough version
-    """
-    from distutils.version import LooseVersion
+# Each time nilearn is imported, check module dependencies
```

Comment:
I find the "each time" wording a bit misleading I would use "when" (The way the import mechanism works in python if you do import nilearn twice the code in `nilearn/__init__.py` is actually executed only once). TBH that's another case of small added value comment if you ask me: we are in `nilearn/__init__.py` so it's quite clear this code is executed when nilearn is imported and the function has a self-explanatory name.